website/docs/tutorials/07-testing/README.md

```diff
@@ -140,15 +138,74 @@ func test_tested_property()
 
 The setup case hook is bound to a matching test case and is executed just before the test case
 itself.
-Use case hooks to configure the behavior of particular test case,
-for example, by calling the [`max_examples`](./02-cheatcodes/max-examples.md) cheatcode.
+These hooks are executed with a context built by the `__setup__` hook, but in isolation for each
+test separately, before jumping into test case function itself.
```

Comment:
- It's not obvious what `these` refers to, because the previous sentence uses the singular form.
- This sentence could be split in two.

Reply:
6f5132f